### ISSUE TITLE
RDF parsing issues

### ISSUE THREAD

**Issue by balhoff** [ExposuresProvider/noctua-models#7]
In some cases there are missing periods needed to terminate triples, for example:

```
riot --validate models/aop_1.ttl 
14:48:39 ERROR riot            :: [line: 19, col: 2 ] Triples not terminated by DOT
```

**Comment by balhoff** [comment]
@mam288 Jena `riot` is a good tool to use to check for stuff like this.

**Comment by balhoff** [comment]
Fixed by #9.
